src/subcommands/build.rs

```diff
@@ -189,6 +191,14 @@ fn build_rust_app(target: &Target, opts: &BuildOptions) -> Result<()> {
     Ok(())
 }
 
+/// Remove a trailing newline from a byte string.
+fn strip_trailing_newline(mut input: Vec<u8>) -> Vec<u8> {
```

Comment:
a helper function goes below the method that uses it

Reply:
Huh, ok. Funky convention; makes you zigzag when you read. :-1: